pallets/capacity/src/lib.rs

```diff
@@ -1141,6 +1144,21 @@ impl<T: Config> StakingRewardsProvider<T> for Pallet<T> {
 		Ok(per_era.saturating_mul(num_eras.into()))
 	}
 
+	// Calculate a reward for one era using the chosen economic model formula + values
+	fn staking_reward_for_era(
+		amount_staked: BalanceOf<T>,
+		total_staked: BalanceOf<T>,
+		reward_pool_size: BalanceOf<T>,
+	) -> BalanceOf<T> {
+		let capped_reward = T::RewardPercentCap::get().mul(amount_staked);
+
+		let proportional_reward = reward_pool_size
+			.saturating_mul(amount_staked)
+			.checked_div(&total_staked)
+			.unwrap_or_else(|| Zero::zero());
```

Comment:
This is going to do integer division and if we would like to round to closer integer maybe we need to use something  else or implement it differently.

Reply:
let's chat about that

Reply:
ok, Wil was the deciding factor of not caring about rounding down a planck, so it'll stay the way it is & I added a note in our chat about the exact method.